src/test/ui/issues/issue-49934.rs

```diff
@@ -23,20 +25,19 @@ fn main() {
     println!("Hello, world!");
 
     // fold_stmt (Semi)
-    #[derive(Debug)] //~ WARN unused attribute
+    #[derive(Debug)]
     "Hello, world!";
 
     // fold_stmt (Local)
-    #[derive(Debug)] //~ WARN unused attribute
+    #[derive(Debug)]
```

Comment:
Now sure if this is the desired behavior

Reply:
The new errors (in the generic parameter position, etc) are expected/desired, the warnings should reappear if they are moved into a separate test.